build-setup-nolog.sh

```diff
@@ -97,82 +97,34 @@ fi
 #   the user to rerun this script without --fast
 # 2) If fast was not specified, but the toolchain from source
 if [ "$IS_LIBRARY" = true ]; then
-    target_toolchain_dir=$RDIR/../../toolchains/
+    target_toolchain_dir=$RDIR/../../toolchains
```

Comment:
I think this is more of a nit, but the rest of the code is cleaned up if you make this a "chipyard_dir".

Reply:
I agree. fixed